### ISSUE TITLE
initial_directory does not work on get_directory_path

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [flet-dev/flet#1184]
**Description**

initial_directory does not work on get_directory_path

**Code example to reproduce the issue:**

```python
on_click=lambda _: file_picker.get_directory_path(initial_directory=os.path.join(os.getcwd(), 'data'))
```

**Describe the results you received:**

**Describe the results you expected:**

Selection directory set to specified dir not to last picked dir. Works with file picking but not folder picking

**Additional information you deem important (e.g. issue happens only occasionally):**

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.4.1
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page:
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: 
Requires: flet-core, httpx, oauthlib, packaging, watchdog, websocket-client, websockets
Required-by:
```

**Operating system**:

Windows 10

**Additional environment details:**

Python3.10

**Comment by FeodorFitsner** [comment]
Duplicate of https://github.com/flet-dev/flet/issues/884
